packages/docs/docs/guides/docsqa.md

```diff
@@ -0,0 +1,82 @@
+# Grounding Answers with a Source of Truth - DocsQA
```

Comment:
I think this would read cleaner in the sidebar if we flipped the order:

```suggestion
# DocsQA: Grounding Answers with a Source of Truth
```

![image](https://github.com/fixie-ai/ai-jsx/assets/829827/8b517a44-9ab0-4722-970f-72b1029a4ccf)